test/test-fixture.html

```diff
@@ -60,22 +60,32 @@
       <x-custom></x-custom>
     </template>
   </test-fixture>
+  <test-fixture id="NestedRepeatFixture">
+    <template>
+      <template id="repeater" is="dom-repeat">
+        <div>{{item}}</div>
+      </template>
```

Comment:
It does require items. Bug happens before items are even generated, template-repeater gets an empty template. Trust me, importNode is the culpit.